Formula/grpc.rb

```diff
@@ -0,0 +1,44 @@
+class Grpc < Formula
+  desc "Google's open-source RPC framework, based on protocol buffers."
+  homepage "http://www.grpc.io/"
+  head "https://github.com/grpc/grpc.git"
+
+  stable do
```

Comment:
The dependency on `sed` is only required on stable, not on HEAD (and presumably future releases). So, what's the preferred syntax for that? I tried moving the checksum and url to the formula body, but then the audit system complained that, as long as there was a stable block, the url and checksum should be there.